.github/workflows/package-Extensions.PersistentStorage.Abstractions.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       run: dotnet build src/${{env.PROJECT}} --configuration Release --no-restore -p:Deterministic=true
 
     - name: dotnet pack ${{env.PROJECT}}
-      run: dotnet pack src/${{env.PROJECT}} --configuration Release --no-build
+      run: dotnet pack src/${{env.PROJECT}} --configuration Release  run: dotnet pack src/${{env.PROJECT}} --configuration Release #--no-build <- OpenTelemetry.Extensions has a conditional net6.0 target which causes dotnet pack to break when -no-build is used (for some reason)
```

Comment:
I think you have mistakenly added this again:
`run: dotnet pack src/${{env.PROJECT}} --configuration Release`